src/fsharp/range.fs

```diff
@@ -163,6 +167,16 @@ type range(code:int64) =
     member m.StartRange = range (m.FileIndex, m.Start, m.Start)
     member m.EndRange = range (m.FileIndex, m.End, m.End)
     member r.FileName = fileOfFileIndex r.FileIndex
+#if DEBUG
+    member r.DebugCode =
+        let endCol = r.EndColumn - 1
+        let startCol = r.StartColumn - 1
+        File.ReadAllLines(r.FileName)
```

Comment:
OK, but put a try-with around this in case the file has changed

Reply:
Actually I has exceptions, because the compiler sets "/temp/unknown.fs" if it doesn't know any better.
The debugger is handling the exceptions just fine.

Reply:
To elaborate: You will see `(0,0-0,0) /temp/unknown IsSynthetic=false -> FileNotFoundException...` in the debugger which is what I'd expect.

@dsyme Do you have any other reason in mind to catch the exception here?

Reply:
Just do it explicitly - we don't know what other debuggers (e.g. mono/xamarin) will do